diff --git a/node_modules/@loadmill/executer/dist/step-handlers/playwright/constants.js b/node_modules/@loadmill/executer/dist/step-handlers/playwright/constants.js
index 6c47391..2581d60 100644
--- a/node_modules/@loadmill/executer/dist/step-handlers/playwright/constants.js
+++ b/node_modules/@loadmill/executer/dist/step-handlers/playwright/constants.js
@@ -27,8 +27,8 @@ exports.validHtmlReportFileTypes = [
     '.zip',
     '.har',
 ];
-exports.MAX_ATTACHMENTS_SIZE_PER_STEP = 1 * 1024 * 1024; // 1 MB
-exports.MAX_HAR_SIZE_PER_STEP = 10 * exports.MAX_ATTACHMENTS_SIZE_PER_STEP; // 10 MB
+exports.MAX_ATTACHMENTS_SIZE_PER_STEP = 10 * 1024 * 1024; // 10 MB
+exports.MAX_HAR_SIZE_PER_STEP = 10 * exports.MAX_ATTACHMENTS_SIZE_PER_STEP; // 100 MB
 var apiPath = env_1.default.CALLBACK_URL + '/api';
 exports.playwrightAttachmentsPath = "".concat(apiPath, "/").concat(url_parts_1.playwrightAttachmentsPart, "/");
 //# sourceMappingURL=constants.js.map
\ No newline at end of file
